applications/DEM_application/custom_utilities/inlet.cpp

```diff
@@ -662,5 +661,18 @@ namespace Kratos {
         mMassInjected[i] += r_cluster.GetMass();
     }
 
+    double DEM_Inlet::GetInputNumberOfParticles(const ModelPart& mp)
+    {
+        double num_part_surface_time = mp[INLET_NUMBER_OF_PARTICLES];
+
+        if (num_part_surface_time >= 0){
+           return num_part_surface_time;
+        }
+
+        else {
+            KRATOS_THROW_ERROR(std::runtime_error, "The value of the Model Part variable INLET_NUMBER_OF_PARTICLES is not a positive int: ", num_part_surface_time);
```

Comment:
I have seen in other PRs that the recommended sintax here is KRATOS_ERROR<<"String" ...
KRATOS_THROW_ERROR is the old style, we have to change some of them.